packages/components/socialFeed/NewsFeed/MapComponent/styles.css

```diff
@@ -0,0 +1,18 @@
+.leaflet-container {
+  height: 100%;
+  width: 100%;
+  border-radius: 1em;
+}
+
+.cluster-icon {
```

Comment:
hmm, why like this?
Shouldn't they be part of the react native implementation?

Reply:
this styles are used in map package component,  so that's why he had added this file,
![Uploading Screenshot 2024-03-08 at 4.15.39 PM.png…]()